config/200-clusterrole.yaml

```diff
@@ -96,4 +99,7 @@ metadata:
 rules:
   - apiGroups: ["triggers.tekton.dev"]
     resources: ["clustertriggerbindings", "clusterinterceptors"]
-    verbs: ["get", "list", "watch"]
\ No newline at end of file
+    verbs: ["get", "list", "watch"]
```

Comment:
Do we need it for both `eventlistener-roles` and `eventlistener-clusterroles`?

Reply:
Verified that adding `events` to `eventlistener-roles` would work